browser/ui/views/speedreader/speedreader_icon_view.cc

```diff
@@ -0,0 +1,160 @@
+// Copyright (c) 2019 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "brave/browser/ui/views/speedreader/speedreader_icon_view.h"
+
+#include <string>
+
+#include "base/notreached.h"
+#include "base/strings/utf_string_conversions.h"
+#include "brave/app/brave_command_ids.h"
+#include "brave/app/vector_icons/vector_icons.h"
+#include "brave/browser/speedreader/speedreader_tab_helper.h"
+#include "brave/grit/brave_generated_resources.h"
+#include "chrome/app/chrome_command_ids.h"
+#include "chrome/browser/themes/theme_properties.h"
+#include "chrome/browser/ui/views/location_bar/location_bar_bubble_delegate_view.h"
+#include "chrome/browser/ui/views/page_action/page_action_icon_view.h"
+#include "chrome/grit/generated_resources.h"
+#include "components/dom_distiller/content/browser/distillable_page_utils.h"
+#include "components/dom_distiller/content/browser/uma_helper.h"
```

Comment:
is it used? pls double-check for unused includes

Reply:
fixed